hiredis.h

```diff
@@ -234,12 +234,22 @@ redisContext *redisConnectUnixWithTimeout(const char *path, const struct timeval
 redisContext *redisConnectUnixNonBlock(const char *path);
 redisContext *redisConnectFd(redisFD fd);
 
+/**
+ *  Redis SSL mode
+ */
+enum redisSslMode {
+    HIREDIS_SSL_VERIFY_NONE = 0,
+    HIREDIS_SSL_VERIFY_PEER,
+    HIREDIS_SSL_VERIFY_FAIL_IF_NO_PEER_CERT,
+    HIREDIS_SSL_VERIFY_CLIENT_ONCE
+};
+
 /**
  * Secure the connection using SSL. This should be done before any command is
  * executed on the connection.
  */
-int redisSecureConnection(redisContext *c, const char *capath, const char *certpath,
-                          const char *keypath, const char *servername);
+int redisSecureConnection(redisContext *c, const char *caPath, const char *certPath,
```

Comment:
Please don't change the naming conventions for argument names. If you'll notice, this is the convention for argument names in other functions as well..

Reply:
Sure, let me fix it.